bottlerocket-settings-models/settings-models/src/lib.rs

```diff
@@ -27,6 +27,7 @@ pub use bottlerocket_string_impls_for as string_impls_for;
 pub use crate::boot::BootSettingsV1;
 pub use settings_extension_autoscaling::{self, AutoScalingSettingsV1};
 pub use settings_extension_aws::{self, AwsSettingsV1};
+pub use settings_extension_bootstrap_commands::{self, BootstrapCommandsSettingsV1, ValidCommand};
```

Comment:
I wouldn't hoist `ValidCommand` to the public API here. `self` is added here so that callers can opt to reach in and grab additional models as needed.

Reply:
Just to add to GitHub something we chatted about after I took a quick look at https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/62 -- there was some feedback about moving `ValidCommand` into the settings extension. The `bootstrap-command` agent in Bottlerocket must parse commands from its config file. Using the models here makes sense to validate the input, and we'd like to avoid having any dependencies between the agent and the extension directly.

This may make it wise to move this one struct back into the `modeled-types`, perhaps as something like `ApiclientCommand` so that it is more generic.